step01/models.py

```diff
@@ -0,0 +1,67 @@
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, String, Integer, ForeignKey
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
+
+
+class Images(Base):
```

Comment:
I think we need to review our data model. Because we're guaranteed to only have one image per thumbnail and one thumbnail per image (one-to-one), there's no need to have two separate models for thumbnail and image. This will help you avoid the backrefs you're using to associate the two models with `back_populate`. 

If you would like to cover how to model relational data let me know.

Reply:
Okay. I remember you mentioning this, but wasn't sure if you wanted to do it yet.
I think I have an understanding of the models relationship.

I will remove the `Thumbnails` model. 
Would you say that the `thumbnail_key` is a sufficient field in the `Images` model to relate to the thumbnails created?